PWGHF/TableProducer/candidateSelectorXicToPKPi.cxx

```diff
@@ -35,11 +35,38 @@
 using namespace o2;
 using namespace o2::analysis;
 using namespace o2::framework;
+//---- temporary MSE function
+float MSE(std::vector<float> y_true, std::vector<float> y_pred, std::vector<float> ScaleMin, std::vector<float> ScaleMax, bool scaling)
+{
+  float mse = 0.0f;
+  float sum = 0.0f;
+
+  std::vector<float> yf_true;
+  if (y_true.size() != y_pred.size()) {
+    LOG(debug) << "size of input vector =" << y_true.size();
+    LOG(debug) << "size of AE output vector =" << y_pred.size();
+    LOG(fatal) << "vectors of input and predictions don't have the same size";
+  } else {                                       // MSE
+    for (size_t j = 0; j < y_pred.size(); ++j) { // for over the features
+      if (scaling == true) {                     // MinMax scaling
+        LOG(debug) << "--------------> MinMax scaling Debug \t" << ScaleMin.at(j) << "\t" << ScaleMax.at(j);
+        yf_true.push_back((y_true.at(j) - ScaleMin.at(j)) / (ScaleMax.at(j) - ScaleMin.at(j)));
+        LOG(debug) << "feature = " << j << " ----> input = " << y_true.at(j) << " scaled feature = " << yf_true.at(j);
+      } else {
+        yf_true.push_back(y_true.at(j));
+      }
+      sum += pow(((yf_true).at(j) - (y_pred).at(j)), 2); // has dimensions
+      LOG(debug) << "feature = " << j << " ----> input = " << yf_true.at(j) << " AE prediction = " << y_pred.at(j);
+    }
+    mse = sum / y_pred.size(); // MSE of a candidate
+  }
```

Comment:
Since this is not HF specific, I would move this in the ML folder. You could consider to add it as a function of the https://github.com/AliceO2Group/O2Physics/blob/master/Tools/ML/MlResponse.h class, or to add a header file with utils functions.
Please also follow the naming conventions of O2Physics when you move it (camel case), thanks!